botocore/signers.py

```diff
@@ -656,14 +656,18 @@ def generate_presigned_url(
         raise UnknownClientMethodError(method_name=client_method)
 
     operation_model = self.meta.service_model.operation_model(operation_name)
+    params = self._emit_api_params(
+        api_params=params,
+        operation_model=operation_model,
+        context=context,
+    )
     bucket_is_arn = ArnParser.is_arn(params.get('Bucket', ''))
     endpoint_url, additional_headers = self._resolve_endpoint_ruleset(
         operation_model,
         params,
         context,
         ignore_signing_region=(not bucket_is_arn),
     )
-
```

Comment:
Was this also black?

Reply:
No. Brought the two empty lines back in d8ed89d.